javascripts/api/api/NonprofitApi.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * API title
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: 0.0.1
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
```

Comment:
If we want to generate and add more endpoints, what is the process for that?

Reply:
> If we want to generate and add more endpoints, what is the process for that?

For new API endpoints, you should put them under the `app/controllers/api_new` directory.